python/vyos/airbag.py

```diff
@@ -124,29 +118,45 @@ def InterceptingException(excepthook,_singleton=[False]):
 # running testing so we are checking that we are on the router
 # as otherwise it prevents dpkg-buildpackage to work
 if get_version() and insession:
-    InterceptingLogger('/run/systemd/journal/dev-log')
+    InterceptingLogger()
     InterceptingException(intercepter)
 
 
 # Messages to print
+# if the key before the value has not time, syslog takes that as the source of the message
 
 FAULT = """\
-Date:       {date}
-VyOS image: {version}
+Report Time:      {date}
```

Comment:
Why duplicate this code and not use the `show system version` output? That would give us single source

Reply:
I have refactored the function which gets the dict but the output is slightly different.
If there is no space in the text, the the first word is taken by syslog to be the service name:

This is why the code has "Report Time" (and not Date) anymore and "Image Version" (and not Version).

```
Apr 20 12:07:38 vyos Date[25010]:       2020-04-20 12:07:38
Apr 20 12:07:38 vyos python3[25010]: VyOS image: 1.3-rolling-202004180117
Apr 20 12:07:38 vyos python3[25010]: Traceback (most recent call last):
Apr 20 12:07:38 vyos python3[25010]:   File "/usr/libexec/vyos/conf_mode/interfaces-dummy.py", line 138, in <module>
Apr 20 12:07:38 vyos python3[25010]:     apply(c)
Apr 20 12:07:38 vyos python3[25010]:   File "/usr/libexec/vyos/conf_mode/interfaces-dummy.py", line 103, in apply
Apr 20 12:07:38 vyos python3[25010]:     raise RuntimeError('test')
Apr 20 12:07:38 vyos RuntimeError[25010]: test
```